exampleCourse/questions/demo/markdownEditorLivePreview/question.html

```diff
@@ -9,5 +9,6 @@
 </pl-question-panel>
 
 <pl-submission-panel>
+  <pl-xss-safe submitted-file-name="ans.md" language="markdown"></pl-xss-safe>
```

Comment:
In addition to this demo question, should we also add `questions/element/xssSafe` to demonstrate it in isolation?

Reply:
Probably a good idea, maybe in combination with a `pl-file-upload` and with different options.

Reply:
That would be great.